lib/BloqadeSchema/test/execute.jl

```diff
@@ -6,16 +6,50 @@ using JSON
 
 
 @testset "to_schema" begin
-    T = 4
-    atoms = [(i,0) for i in 1:10]
+    T = 2
+    atoms = [(π*i,0) for i in 1:10]
+
+    # values = [1.0,nothing,Waveform(t->sin(2π*t/T)^2,T/2)]
+    values = [1.0,constant(;duration=T,value=1.0),Waveform(t->sin(2π*t/T)^2,T)]
+    params = BloqadeSchema.SchemaConversionParams()
+
+    check_atom_res = x->all(BloqadeSchema.check_resolution.(params.atom_position_resolution,x))
```

Comment:
```suggestion
check_atom_res(x) = all(BloqadeSchema.check_resolution.(params.atom_position_resolution,x))
```

use a named function is usually preferred if possible since it gives clearer error msg when error